crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -845,6 +907,7 @@ pub async fn run(
             docker,
             output_dir,
         } => {
+            telem(TelemetryEvent::KeyVal(KeyVal::command("prune")));
```

Comment:
tiniest nit: this feels like a good target for a macro

Reply:
I'm still not sold on this api, but when I am I will macro it up!